src/AWSS3.jl

```diff
@@ -218,6 +237,36 @@ function s3_exists(
     end
 end
 
+"""
+    s3_exists_unversioned([::AbstractAWSConfig], bucket, path; kwargs...)
+
+Returns a boolean whether an object exists at  `path` in `bucket`.
+
+See [`s3_exists_versioned`](@ref) to check for specific versions.
+"""
+function s3_exists_unversioned(aws::AbstractAWSConfig, bucket, path; kw...)
+    return (endswith(path, "/") ? _s3_exists_dir : _s3_exists_file)(
+        aws, bucket, path; kw...
+    )
+end
+
+"""
+    s3_exists([::AbstractAWSConfig], bucket, path; version=nothing, kwargs...)
```

Comment:
It seems the `kwargs` only do anything in the versioned case in which case they get passed to `s3_get_meta` where it gets forwarded to `head_object` where they are apparently not accepted: https://github.com/JuliaCloud/AWS.jl/blob/56febe52eb7754df869658dbaefd9519efc1aa13/src/services/s3.jl#L2616

Let’s just remove all these `kwargs…` from these exist calls, since it seems impossible to use them for anything.